smriprep/interfaces/freesurfer.py

```diff
@@ -0,0 +1,176 @@
+import os
+from nipype import logging
+from nipype.utils.filemanip import check_depends
+from nipype.interfaces.base import traits, InputMultiObject, isdefined
+from nipype.interfaces import freesurfer as fs
+
+iflogger = logging.getLogger('nipype.interface')
+
+
+class ReconAllInputSpec(fs.preprocess.ReconAllInputSpec):
+    directive = traits.Enum(
+        'all',
+        'autorecon1',
+        # autorecon2 variants
+        'autorecon2',
+        'autorecon2-volonly',
+        'autorecon2-perhemi',
+        'autorecon2-inflate1',
+        'autorecon2-cp',
+        'autorecon2-wm',
+        # autorecon3 variants
+        'autorecon3',
+        'autorecon3-T2pial',
+        # Mix of autorecon2 and autorecon3 steps
+        'autorecon-pial',
+        'autorecon-hemi',
+        # Not "multi-stage flags"
+        'localGI',
+        'qcache',
+        argstr='-%s',
+        desc='process directive',
+        usedefault=True,
+        xor=['steps'],
+        position=0)
+    steps = InputMultiObject(
+        traits.Enum(
+            # autorecon1
+            'motioncor',
+            'talairach',
+            'nuintensitycor',
+            'normalization',
+            'skullstrip',
+            # autorecon2-volonly
+            'gcareg',
+            'canorm',
+            'careg',
+            'careginv',     # 5.3
+            'rmneck',       # 5.3
+            'skull-lta',    # 5.3
+            'calabel',
+            'normalization2',
+            'maskbfs',
+            'segmentation',
+            # autorecon2 per-hemi
+            'tessellate',
+            'smooth1',
+            'inflate1',
+            'qsphere',
+            'fix',
+            'white',
+            'smooth2',
+            'inflate2',
+            'curvHK',       # 6.0
+            'curvstats',
+            # autorecon3 per-hemi
+            'sphere',
+            'surfreg',
+            'jacobian_white',
+            'avgcurv',
+            'cortparc',
+            'pial',
+            'pctsurfcon',
+            'parcstats',
+            'cortparc2',
+            'parcstats2',
+            'cortparc3',
+            'parcstats3',
+            'label-exvivo-ec',
+            # autorecon vol
+            'cortribbon',
+            'hyporelabel',  # 6.0
+            'segstats',
+            'aparc2aseg',
+            'apas2aseg',    # 6.0
+            'wmparc',
+            'balabels',
+            ),
+        desc='specific process directives',
+        xor=['directive'],
+        position=0)
+
+
+class ReconAll(fs.ReconAll):
+    input_spec = ReconAllInputSpec
+
+    def _is_resuming(self):
+        subjects_dir = self.inputs.subjects_dir
+        if not isdefined(subjects_dir):
+            subjects_dir = self._gen_subjects_dir()
+        if os.path.isdir(
+                os.path.join(subjects_dir, self.inputs.subject_id, 'mri')):
+            return True
+        return False
```

Comment:
I don't think this changed... I think I just failed to remove it. Will do before merge, but shouldn't affect review.